### DISCUSSION TITLE
Windows Vista Flip3D Switch Feature

### DISCUSSION THREAD

**Discussion by smtsoetbeer** [valinet/ExplorerPatcher#1381] [Ideas]
Is it Possible to integrate the Flip3D Design for App Switcher?

**Comment by XuanChen1234** [comment]
![3D Aero Flip](https://github.com/valinet/ExplorerPatcher/assets/140480110/8ba13f73-ef94-476d-9067-5bef9ee84b04)

**Comment by XuanChen1234** [comment]
If it is not planned yet, I hope this will be a thing in the next update.

**Comment by XuanChen1234** [comment]
@Amrsatrio Ask for the option to add 3D Aero Flip switcher.

**Reply by pyrates999** [reply]
Code no longer exists, so this is not possible.
